include/networkit/sparsification/SimmelianScore.hpp

```diff
@@ -83,8 +84,8 @@ class SimmelianScore : public EdgeScore<double> {
 
     void matchNeighbors(node ego, node alter, bool reciprocityCheck,
                         std::vector<RankedEdge>::const_iterator &egoIt,
-                        const RankedNeighbors &egoNeighbors, std::set<node> &egoNeighborsUnmatched,
-                        std::set<node> &alterNeighborsUnmatched, count rank, count &overlap);
+                        const RankedNeighbors &egoNeighbors, std::unordered_set<node> &egoNeighborsUnmatched,
+                        std::unordered_set<node> &alterNeighborsUnmatched, count rank, count &overlap);
```

Comment:
Doesn't this affect users of the function? If yes, deprecation (or maybe just providing two functions) is necessary.